src/axom/primal/tests/primal_clip.cpp

```diff
@@ -83,7 +84,10 @@ TEST(primal_clip, simple_clip)
     PolygonType poly = axom::primal::clip(tri, bbox);
     EXPECT_EQ(4, poly.numVertices());
 
-    EXPECT_EQ(PointType(.5), poly.vertexMean());
+    for(int dim = 0; dim < 3; ++dim)
+    {
+      EXPECT_NEAR(PointType(0.5)[dim], poly.vertexMean()[dim], EPS);
```

Comment:
Based purely on the line of code, I think the following would be equivalent and simpler:
```suggestion
      EXPECT_NEAR(0.5, poly.vertexMean()[dim], EPS);
```